package.json

```diff
@@ -10,7 +10,8 @@
     "backbone.paginator": "~2.0.3",
     "coffee-loader": "^0.7.3",
     "coffee-script": "1.6.1",
-    "@edx/studio-frontend": "0.1.0",
+    "css-loader": "^0.28.5",
+    "@edx/studio-frontend": "0.1.1",
```

Comment:
This needs to be modified to point to a hash from https://github.com/edx/studio-frontend/pull/31 before provisioning; the `devDependencies` -> `dependencies` changes there are required for the webpack config changes here to not error out.